### DISCUSSION TITLE
Is there any chance to get Boris working on Apple Silicon Macs (M2)?

### DISCUSSION THREAD

**Discussion by madamadami** [olivierfriard/BORIS#696] [Q&A]
Dear Community,
we're desperately looking for a way to make Boris running on Apple Silicon computers. Preferably with free software only. Could someone point out an approach? I tried Docker Desktop with the command line script, but the loading of Boris got stuck at the logo (eye with a platypus). Find command line output attached to this post.

Since a lot of students in our teams use Macs with Apple Silicon,  help is much appreciated!

[command line output.txt](https://github.com/olivierfriard/BORIS/files/13708656/command.line.output.txt)

**Comment by olivierfriard** [comment]
Hello,

The BORIS docker seems to have some issues (see issue #694).
Unfortunately, I cannot work now on developing a BORIS native program on MacOS because I have no access to a physical Mac.

**Reply by madamadami** [reply]
What a pity!

**Comment by Philippe-s-Levi** [comment]
did you try a windows emulator ?

**Reply by madamadami** [reply]
Yes, I tried it with Parallel Desktop and it worked!

**Comment by olivierfriard** [comment]
Hi,
I just released a version of BORIS that should run natively on macOS with some limitations described on https://www.boris.unito.it/download_mac.
This is an experimental version so test it and let me know
